gulp/tasks/copy.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var changed = require( 'gulp-changed' );
+var replace = require( 'gulp-replace' );
+var config = require( '../config' ).copy;
```

Comment:
Is that necessary? You're already in the `copy.js` file, it's kinda obvious what the `config` var is pointing to right?